app/controllers/challenge_claims_controller.rb

```diff
@@ -147,4 +147,7 @@ def challenge_claim_params
     )
   end
 
+  def user_scoped?
+    params[:for_user].to_s.casecmp("true").zero?
```

Comment:
Do we actually care what the value of the parameter is, or just that it's present? As far as I can tell, we don't currently have different behavior for `?for_user=testy` than `?for_user=true` -- it always shows a logged in user their own claims.

(I'm not advocating for it's removal; just wondering if I'm missing something.)

Reply:
Rails doesn't care if it's "true" or some variants. Here's the method it uses for turning these values into booleans:

https://github.com/rails/rails/blob/5-1-stable/activemodel/lib/active_model/type/boolean.rb

That doesn't happen here because we're not setting a boolean field on a model, but we can use `cast` if we like:

https://github.com/rails/rails/blob/663206d20aec374a28a24bb43bc7b1233042ed9b/activemodel/test/cases/type/boolean_test.rb#L8-L24

Reply:
What I was trying to detect was any tampering with the URL.

A non-maintainer user who follows links would only get the link format `for_user=true`, so if we receive anything else, that was someone attempting to "hack" it. It felt wrong to allow someone to manually replace `for_user` with something else, maybe the username of their nemesis, and still get their own claims page.

Reply:
> Rails doesn't care if it's "true" or some variants. Here's the method it uses for turning these values into booleans:
> 
> https://github.com/rails/rails/blob/5-1-stable/activemodel/lib/active_model/type/boolean.rb
> 
> That doesn't happen here because we're not setting a boolean field on a model, but we can use `cast` if we like:
> 
> https://github.com/rails/rails/blob/663206d20aec374a28a24bb43bc7b1233042ed9b/activemodel/test/cases/type/boolean_test.rb#L8-L24

Off-topic, but I was surprised `"no"` wasn't in there. Then I remembered the time when Norway was deemed `false`: https://twitter.com/chrisjrn/status/1232016100038266880